weave

```diff
@@ -471,11 +471,8 @@ case "$COMMAND" in
         fi
         if [ "$1" = "-password" ] ; then
             [ $# -gt 1 ] || usage
-            PASSWORD=$2
-            shift 2
-        elif [ "$1" = "-passwordfile" ] ; then
-            [ $# -gt 1 ] || usage
-            PASSWORDFILE=$2
+            WEAVE_PASSWORD="$2"
+            export WEAVE_PASSWORD
```

Comment:
Well, it would when you `source $(which weave) launch`, maybe that's how
you mixed it up ;)

On Thu, 8 Jan 2015 18:22 rade <notifications@github.com> wrote:

> In weave
> <https://github.com/zettio/weave/pull/332#discussion-diff-22669714>:
>
> > @@ -471,11 +471,8 @@ case "$COMMAND" in
> >          fi
> >          if [ "$1" = "-password" ] ; then
> >              [ $# -gt 1 ] || usage
> > -            PASSWORD=$2
> > -            shift 2
> > -        elif [ "$1" = "-passwordfile" ] ; then
> > -            [ $# -gt 1 ] || usage
> > -            PASSWORDFILE=$2
> > +            WEAVE_PASSWORD="$2"
> > +            export WEAVE_PASSWORD
>
> Ignore me. For some reason I thought export exported to the parent env.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zettio/weave/pull/332/files#r22669714>.
>